### DISCUSSION TITLE
withMaxDepth not available in Trees: One Parent per Node (One-to-Many)

### DISCUSSION THREAD

**Discussion by whoami15** [staudenmeir/laravel-adjacency-list#224] [General]
Hello,

I'm checking the `1.12` branch and noticed that the `withMaxDepth` constraint is only available in **Graphs with Multiple Parents per Node (Many-to-Many)**. Is this intentional?

**Comment by staudenmeir** [comment]
Hi @whoami15,
`withMaxDepth()` for trees is not available in this branch because it depends on another method that's only available in the `master` branch.

**Reply by whoami15** [reply]
Hi @staudenmeir,

Thank you for clarifying. Much appreciated!
